plugin/src/App/Components/PlaySoloAutoConnector.lua

```diff
@@ -0,0 +1,38 @@
+local RunService = game:GetService("RunService")
+
+local Rojo = script:FindFirstAncestor("Rojo")
+
+local Roact = require(Rojo.Roact)
+
+local e = Roact.createElement
+
+local PlaySoloAutoConnector = Roact.Component:extend("PlaySoloAutoConnector")
+
+function PlaySoloAutoConnector:render()
+	return nil
+end
+
+function PlaySoloAutoConnector:didMount()
```

Comment:
I'm not a big fan of this architecture, as it uses prop drilling and a component that just calls a function for the app. Instead, we can use the architecture that we now use for many other features- a call from `App:init()` that uses `self:startSession()`